circle.yml

```diff
@@ -1,12 +1,9 @@
 machine:
   node:
-    version: '0.10'
+    version: '0.12'
 dependencies:
-  pre:
-    - sudo apt-get update; sudo apt-get -y install python-pip libglew-dev gdb
-    - sudo pip install awscli
```

Comment:
No, you're right! Want to readd `pip install awscli` and merge? BTW not sure we need sudo there (http://stackoverflow.com/a/29113236/244789)